test/parallel/test-intl-v8BreakIterator.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+require('../common');
+var assert = require('assert');
+
+if (global.Intl != undefined &&
+   Intl.v8BreakIterator != undefined) {
+  try {
+    new Intl.v8BreakIterator();
+  } catch (e) {
+    // OK- exception is OK, crash is not OK.
+    assert(e.message.indexOf('ICU data') !== -1,
+      'Unexpected error: ' + e.message);
+  }
+} else {
+  console.log('1..0 # Skipped: no Intl');
```

Comment:
Not exactly wrong but it's more idiomatic in the test suite to put this at the top, i.e.:
```js
if (global.Intl === undefined || Intl.v8BreakIterator === undefined) {
  console.log('1..0 # Skipped: no Intl');
  return;
}
```